public/js/index.js

```diff
@@ -15,3 +15,5 @@ function toggleDraft() {
     x.innerHTML = 'View More...';
   }
 }
+
+console.log(messages);
```

Comment:
remove this

Reply:
Whoops, sorry. I thought I removed those already.